README.md

```diff
@@ -9,8 +9,8 @@ Thanks for everything! See you next game.
 
 ## Requirements
 
-- Node 19+
-- Python 3.9+ (optional - for refreshing the data store)
+-   Node 19+
```

Comment:
Turns out, this is how Prettier wants it. Currently a [contentious issue](https://github.com/prettier/prettier/issues/5019).